db/migrate/20151208150956_fix_host_storage_replication_on_upgrade.rb

```diff
@@ -9,36 +8,10 @@ class HostsStorage < ActiveRecord::Base
   end
 
   def up
-    if on_replication_target?
-      run_for_replication_target
-    else
-      run_for_replication_source
-    end
+    HostsStorage.delete_all if on_replication_target?
```

Comment:
I think this still has to happen because if we run through https://github.com/ManageIQ/manageiq/blob/master/db/migrate/20150921204114_add_vmware_ro_datastores_to_hosts_storages.rb we will still have incorrect ids. I'm just not sure how to deal with it now that the remote servers won't automatically sync the rows back up.

I will need to look into `pglogical.alter_subscription_synchronize` to see if that could solve our problems. (Docs can be found [here](https://2ndquadrant.com/en/resources/pglogical/pglogical-docs/))

Alternatively we can enforce that people don't try to upgrade their global dbs because of this. Thoughts?